app/services/workflow.py

```diff
@@ -57,8 +57,11 @@ async def run_workflow(
             "MODEL_NAMES": ",".join(model_names),
             "DATASET_NAMES": ",".join(dataset_names),
             "METRICS": ",".join(experiment.metrics),
+            "HF_HOME": ".",  # TODO not sure whether i should define it here...
```

Comment:
I don't recommend defining it here. Not every experiment template will rely on HF